### PR TITLE
1609 release standard images

### PR DESCRIPTION
**Pull request by jimmykarily**
Part of #1609

### CODE REVIEW

File: .github/kubernetes-versions.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[
+    "v1.25.11+k3s1",
+    "v1.26.6+k3s1",
+    "v1.27.3+k3s1"
```
Comment (jimmykarily) [inline .github/kubernetes-versions.json:4]: These should match what the repository provides: https://github.com/kairos-io/packages/blob/main/packages/k8s/k3s/collection.yaml

I wonder if we should parse the `collection.yaml` from the remote location directly :thinking: . This way, when we want to change the versions we support, we will only have to change the versions built in the packages repository and this pipeline will pick it up automatically.

Comment (Itxaka) [reply .github/kubernetes-versions.json:4]: I would say yes, otherwise we will need to do manual stuff no?

**Comment by jimmykarily** [comment]
This PR needs to also migrate the release-arm workflow from provider-kairos repo

File: .github/kubernetes-versions.json (hunk 1 above)
Comment (jimmykarily) [reply .github/kubernetes-versions.json]: Done (I think, we will know when the pipeline runs)

Comment (jimmykarily) [reply .github/kubernetes-versions.json]: https://github.com/kairos-io/kairos/pull/1645/files#diff-e426ed45842837026e10e66af23d9c7077e89eacbe6958ce7cb991130ad05adaR40-R42

**Comment by jimmykarily** [comment]
Closing this draft to save some CI resources. I still have quite some things to push.

**Pull request closed without merging**